linodecli/cli.py

```diff
@@ -42,403 +39,43 @@ def __init__(self, version, base_url, skip_config=False):
-
-                        # handle arrays
-                        if "item_type" in info:
-                            new_arg.arg_item_type = info["item_type"]
-                        cli_args.append(new_arg)
-
-                    # looks for param names that will be obscured by args
-                    # clone the params since they're shared by all methods in this
-                    # path, and we only want to modify this method's params
-                    use_params = [c.clone() for c in params]
-                    use_path = path
-                    for p in use_params:
-                        if p.name in args:
-                            # or (m == 'get' and p.name in model_attrs):
-                            # if we found a parameter name that is also and argument name
-                            # append an underscore to both the parameter name and the
-                            # parameter name in the URL
-                            use_path = use_path.replace(
-                                "{" + p.name + "}", "{" + p.name + "_}"
-                            )
-                            p.name += "_"
-
-                    self.ops[command][action] = CLIOperation(
-                        command,
-                        action,
-                        m,
-                        use_path,
-                        summary,
-                        cli_args,
-                        response_model,
-                        use_params,
-                        use_servers,
-                        docs_url=docs_url,
-                        allowed_defaults=allowed_defaults,
-                        action_aliases=action_aliases,
-                    )
-
-        # remove any empty commands (those that have no actions)
-        to_remove = []
-        for command, actions in self.ops.items():
-            if len(actions) == 0:
-                to_remove.append(command)
-
-        for command in to_remove:
-            del self.ops[command]
+                operation = getattr(path, m)
+                if operation is None or ext["skip"] in operation.extensions:
+                    continue
+                action = operation.extensions.get(
+                    ext["action"], operation.operationId
+                )
+                if not action:
+                    continue
+                if isinstance(action, list):
+                    action = action[0]
+                if command not in self.ops:
+                    self.ops[command] = {}
+                self.ops[command][action] = OpenAPIOperation(
+                    command, operation, m, path.parameters
+                )
 
         # hide the base_url from the spec away
-        self.ops["_base_url"] = spec["servers"][0]["url"]
-        self.ops["_spec_version"] = spec["info"]["version"]
+        self.ops["_base_url"] = self.spec.servers[0].url
+        self.ops["_spec_version"] = self.spec.info.version
+        self.ops["_spec"] = self.spec
```

Comment:
A question confused me... If we are going to drop these when loading the baked data, why do we have to set them here?

Reply:
Ah never mind, I got it